bindings/python/__init__.py

```diff
@@ -0,0 +1,124 @@
+"""
```

Comment:
Delete this fine. It does nothing

Reply:
The entire commented block of 120 lines?

Reply:
Sorry meant to say file not fine.

Reply:
Ok.